kafkakewl-common/src/main/scala/com/mwam/kafkakewl/common/telemetry/GlobalTracer.scala

```diff
@@ -6,13 +6,22 @@
 
 package com.mwam.kafkakewl.common.telemetry
 
-import io.opentelemetry.api.trace.Tracer
+import io.opentelemetry.api.OpenTelemetry
+import io.opentelemetry.api.trace.{DefaultTracer, Tracer}
 import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk
-import zio.{TaskLayer, ZIO, ZLayer}
+import zio.{TaskLayer, ZIO, ZLayer, System}
 
 object GlobalTracer {
   val live: TaskLayer[Tracer] =
     ZLayer.fromZIO(
-      ZIO.attempt(AutoConfiguredOpenTelemetrySdk.initialize.getOpenTelemetrySdk.getTracer(getClass.getPackageName))
+      {
+        for {
+          _ <- System.env("OTEL_EXPORTER_OTLP_ENDPOINT").flatMap(zio.ZIO.fromOption)
```

Comment:
can you double check if all of these are required

Reply:
also worth checking what env vars exist in the OCP pods? Are all of these set in prod/test?

Reply:
Yes they are all set in test/prod pods. More than here, even. You confirmed offline that these three were sufficient to get tracing working locally.